src/ploomber/sources/notebooksource.py

```diff
@@ -1036,7 +1035,7 @@ def _warn_on_unused_params(nb, params):
                       f'task\'s source code: {pretty_print.iterable(unused)}')
 
 
-def add_parameters_cell(path, extract_upstream, extract_product):
+def add_parameters_cell(path, extract_upstream=False, extract_product=False):
```

Comment:
add a new confition here

```
if not extract_upstream and not extract_product:
    source += '# add default values for parameters here'
```

Reply:
Do we need this? Looks like they might be redundant.
```python
    if extract_upstream:
        source += """\
# declare a list tasks whose products you want to use as inputs
upstream = None
"""
    if extract_product:
        source += """\
# declare a dictionary with the outputs of this task
product = None
"""
```

Reply:
why redudnant? the first two apply when any of them is True or both. this new one applies when none of them are true